src/hotspot/share/opto/machnode.hpp

```diff
@@ -197,7 +197,11 @@ class MachOper : public ResourceObj {
 // ADLC inherit from this class.
 class MachNode : public Node {
 public:
-  MachNode() : Node((uint)0), _num_opnds(0), _opnds(NULL) {
+  MachNode() : Node((uint)0), _num_opnds(0), _opnds(NULL)
+#if INCLUDE_ZGC
+    , _barrier(0)
+#endif
+  {
```

Comment:
I think they can be combined into one line as upstream
```c++
  MachNode() : Node((uint)0), _num_opnds(0), _opnds(NULL) {
```

Reply:
I change it by adding `ZGC_ONLY( _barrier = 0; )` into the method body.
It is a one-line change. It's better than the previous four-line change and two-line (1+ & 1-) change in the original patch.